tests/run_tests.py

```diff
@@ -112,7 +119,7 @@ def _test_python(args):
 
     try:
         from multiprocessing import cpu_count  # pylint: disable=C0415
-        threads = min(8, cpu_count())  # To prevent running out of memory
+        threads = max(8, cpu_count())  # To prevent running out of memory
```

Comment:
NITS, now we explicitly control the number of threads to launch the test with `-t` in the win_test script. So `threads = args.threads or env_threads or threads` will never evaluated to this part. So change here will not increase the test parallelism.

Reply:
Ah, didn't notice it slipped here, will revert it.